esx_service/utils/auth_api.py

```diff
@@ -488,9 +488,26 @@ def vm_in_any_tenant(vms):
 
     return None
 
+
+def named_tenant(func):
+
+    def not_supported():
+        error_info = error_code.generate_error_info(ErrorCode.FEATURE_NOT_SUPPORT,
+                                                    auth_data_const.DEFAULT_TENANT)
+        return error_info
+
+    def wrap_function(name, vm_list):
```

Comment:
I think wrap_function is needed. The call to decorator just pass in the func name. See the example here: https://realpython.com/blog/python/primer-on-python-decorators/